tests/data/Q4185820-en/output.html

```diff
@@ -0,0 +1,90 @@
+<p>
```

Comment:
Do we have \n in the final output? Please remove them, if yes, to reduce the size in production.

Reply:
No, for the snapshot tests they are pretty-printed to make the diffs easier to read.
In the main wikiparser the newlines between elements are not printed.